UofS-Pulse-Binfo/analyzedphenotypes
# Trait importer

## Metadata

 - This PR doesn't address an issue.



- [ ] This PR is dependent upon PR #62

Documentation:

## Description
This importer allows for easy uploading of trait dictionaries. This allows users with large datasets to pre-load the traits making it faster and easier to load phenotypic data for a large number of traits. This also makes it easier to test trait pages ;-p

## Dependencies
This PR is dependent upon #62 because it needs to renamed `include` => `includes` directory in order for the importer to be found.

![screen shot 2019-02-18 at 5 18 09 pm](https://user-images.githubusercontent.com/1566301/52980719-25e49780-33a1-11e9-9629-14d41c924eb1.png)

## Testing?




This importer has been tested both through the interface and using automated testing. To test it yourself:
1. Go to Admin > Tripal > Data Loaders > Phenotypic Traits
2. Follow the instructions on the page to create a trait dictionary and choose an already configured genus.
3. Run the job and check the trait chart or publish the page.

NOTE: Make sure to clear the cache if the importer is not showing up.
